src/sinks/aws_cloudwatch_metrics/mod.rs

```diff
@@ -80,6 +85,7 @@ pub struct CloudWatchMetricsSinkConfig {
     ///
     /// [iam_role]: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html
     #[configurable(deprecated)]
```

Comment:
Can we have a useful deprecated message here?

Reply:
It's `hidden` as well, it hasn't been present in the documentation for some time.